### DISCUSSION TITLE
Introspection command should return sorted results.

### DISCUSSION THREAD

**Discussion by tbrannam** [kamilkisiela/graphql-inspector#1830] [Q&A] [answered]
I'd like to use the introspection command to track schema changes over time in a git repo.   It appears the schema output is likely dependent on the order of the resolvers added.  

To make for a more consistent output - it would be nice if the introspection command dictated some fixed sort order for the output file - this would help with folks that are looking at diffs of the schema over time.

**Comment by kamilkisiela** [comment, marked as answer]
#1831 thank you for the PR!
